cmd/pulumictl/create/docs-build/cli.go

```diff
@@ -29,6 +34,11 @@ type Payload struct {
 	Project          string `json:"project"`
 	ProjectShortname string `json:"project-shortname"`
 	Ref              string `json:"ref"`
+	Category         string `json:"category"`
+	DisplayName      string `json:"display-name"`
+	IsComponent      bool   `json:"is-component"`
```

Comment:
Nit: Could we just go with `Component`?

Reply:
Alternatively, if we anticipate that this could be used for more types of things, might be better to go with a `Type` enum, but breaking changes on an internal tool are no big deal in any case.